content/docs/integrations/aws-resources/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Configuring AWS Resources Autodiscovery
+publishedDate: '2022-03-29T21:00:00.0Z'
+description: How to configure Roadie to read AWS resources
+
+humanName: AWS Resources
+logoImage: '../../../assets/logos/aws/Amazon_Web_Services_Logo.png'
+integrationType: Integration
+---
+
+## Introduction
+
+Roadie has the capability to automatically ingest resources from AWS. This is done by automatic discovery and currently ingest all configured resource types per AWS region.
+
+This guide describes how to set up Roadie to access your specific AWS resources and automatically ingest them.
+
+
+##  Step 1: Get the roadie IAM details
+
+Navigate to `Administration > Settings > AWS Resources` and make a note of the Roadie backend role ARN and account ID. This is mentioned on the title text of the AWS Resources integration settings page.
+
+##  Step 2: Create a federated role in your account for Roadie
+
+Follow the steps [here](/docs/details/accessing-aws-resources) to create the role. 
```

Comment:
There is a cloudformation template and some code to generate the correct input for that somewhere in the roadie code.